docs/en/rst/installing/mysql.rst

```diff
@@ -1,19 +1,19 @@
 .. _mysql:
 
-MySQL
-#####
+MariaDB
+#######
 
-It is strongly advised to use MariaDB which is a drop-in replacement for
-MySQL and is fully compatible with Bugzilla.  If MySQL must be used, be
-aware that the minimum required version is MySQL 5.0.15 and that MySQL 8
-or higher cannot be used at this time.
+It is strongly advised to use MariaDB which is fully compatible with Bugzilla.
 
-All commands in this document work regardless of whether MySQL or
-MariaDB are used.
+If MySQL is used, be aware that the minimum required version is MySQL 5.0.15.
```

Comment:
Minimum version of MySQL in Bugzilla 5.2 is actually 5.6.12
(and minimum MariaDB is 10.0.5)